src/tests/efl_check.h

```diff
@@ -136,7 +136,7 @@ _efl_test_option_disp(int argc, char **argv, const Efl_Test_Case *etc)
           }
         else if (strcmp(argv[i], "--valgrind") == 0)
           {
-	     char *nav = (char *)malloc(sizeof(char) * (argc + 3));
+	           char **nav = malloc(sizeof(char*) * (argc + 3));
```

Comment:
```suggestion
	           char **nav = malloc(sizeof(char*) * (argc + 3));
```

Reply:
I don't know what happened here. When I committed I saw this, but this isn't happening here in my code haha. Thanks

Reply:
Oh, I can't accept your suggestion, the button yields: 
```
This suggestion is invalid because no changes were made to the code.
```